jest/helpers.ts

```diff
@@ -77,7 +80,13 @@ export const makeTemplate = (
     return values[number - 1];
   });
 
-export const cleanup = (directory: string) => rimraf.sync(directory);
+export const cleanup = (directory: string, async = true) => {
```

Comment:
Can we have 2 functions named: `cleanupSync` and `cleanup`. Passing a boolean as a function parameter is a code-smell.